data/json/items/tool/misc.json

```
@@ -43,6 +43,31 @@
     "color": "yellow",
     "flags": [ "WATCH", "ALARMCLOCK" ]
   },
+  {
+    "type": "TOOL",
+    "id": "eac_butane_can",
+    "name": "can of butane",
+    "name_plural": "cans of butane",
+    "description": "A typical can of butane that you would normally use to fill a lighter, buuuuut there might be another use... ",
```

```suggestion
    "description": "A typical can of butane that you would normally use to fill a lighter.  It could be useful as a fuel, but also as a solvent or chemical reagent.",
```

Once this is in game it'll surely develop a number of uses, may as well give it a general description right from the top.